integrationtests/system_test.go

```diff
@@ -0,0 +1,25 @@
+package integrationtests
+
+import (
+	"context"
+	"testing"
+
+	"github.com/kubernetes-csi/csi-proxy/client/api/system/v1alpha1"
+	v1alpha1client "github.com/kubernetes-csi/csi-proxy/client/groups/system/v1alpha1"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetSerialNumber(t *testing.T) {
+	t.Run("GetSerialNumber", func(t *testing.T) {
+		client, err := v1alpha1client.NewClient()
+		require.Nil(t, err)
+		defer client.Close()
+
+		request := &v1alpha1.GetSerialNumberRequest{}
+		response, err := client.GetSerialNumber(context.TODO(), request)
+		require.Nil(t, err)
+		require.NotNil(t, response)
+		require.NotEmpty(t, response.SerialNumber)
+		t.Logf("The serial number is %s", response.SerialNumber)
```

Comment:
Can you also invoke the same cmd - `wmic bios get serialnumber` from the test and compare the value? Ideally we could have queried the serial number from other out-of-band mechanism (like through the hypervisor if possible) but that will make the test specific to a certain hypervisor environment.